pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -47,6 +47,15 @@ const (
 	podCleanupPollFreq = time.Second
 )
 
+const (
+	// SignalEphemeralContainerFsLimit is amount of storage available on filesystem requested by the container
```

Comment:
nit:
s/SignalEphemeralContainerFsLimit/signalEphemeralContainerFsLimit

same for others below

Reply:
Bah, that's what I get for learning to depend on IDE listing.  Fixed